stdpopsim/models.py

```diff
@@ -235,8 +235,12 @@ def simulate(self, contig, samples, seed=None):
         return ts
 
 
-# Resuable generic population
+# Resuable generic populations
 _pop0 = Population(name="pop0", description="Generic population")
+_popAnc = Population(name="popA", description="Generic ancestral population",
```

Comment:
i'm a bit unsure what the scope of these objects is. @apragsdale how do you want these to be available?

Reply:
Yeah good question - I don't know if there's a good reason for them to be defined outside the model classes. I was mainly following how the `PiecewiseConstantSize` model was set up for consistency, and I imagine `_pop0` was meant to be reused if needed (though I didn't, maybe should have). Probably more intuitive to just define the populations within each class as needed, you think?

Reply:
I think it's okay to define these at this level but it would be best to reuse them (e.g. `_pop0`). @jeromekelleher you okay with this bit of ugliness too?

Reply:
Alright - made a few small changes. Let me know what you think @andrewkern, @jeromekelleher , and if anything else could be tidied up